test/Test/Handler.purs

```diff
@@ -65,6 +65,10 @@ testParams = do
         setupMockApp $ use paramsHandler
         sendTestRequest withoutParams assertTestHeaderAbsent
         sendRequest GET urlWithQueryParam id assertTestHeaderExists
+    testExpress "getQueryParams" $ do
+        setupMockApp $ use paramsHandler
+        sendTestRequest withoutParams assertTestHeaderAbsent
+        -- sendRequest GET urlWithQueryParams id assertTestHeaderExists
```

Comment:
I don't how to implement the test on `sendRequest GET urlWithQueryParams id assertTestHeaderExists`

Reply:
No worries, I can write it later.